### PR TITLE
PIC-1743 - Form Validation

### PR DESCRIPTION
**Pull request by theDustRoom**
✨ Included form validation
💄 Included error summary
💄 Included error messages inline
✅ Updated unit tests

Signed-off-by: theDustRoom <paul.massey@digital.justice.gov.uk>

### CODE REVIEW

File: server/views/record-of-oral/risk-assessment.njk [hunk 1]
```diff
@@ -81,7 +92,11 @@
       id: "likelihoodAssessment",
       label: {
         text: "Your assessment"
-      }
+      },
+      value: data.likelihoodAssessment,
+      errorMessage: {
+        text: "Enter the tool name"
```
Comment (katie-cawthorne) [inline server/views/record-of-oral/risk-assessment.njk:98]: Does this error message need to be "Enter your assessment"?

**Review by katie-cawthorne — CHANGES_REQUESTED** [review]
Looks great, just the comment I added about the error message on `risk-assessment.njk`

**Review by sarahyjja — DISMISSED** [review]
LCTM

File: server/views/record-of-oral/risk-assessment.njk (hunk 1 above)
Comment (theDustRoom) [reply server/views/record-of-oral/risk-assessment.njk:98]: Good catch.

File: server/views/record-of-oral/risk-assessment.njk [hunk 2]
```diff
@@ -81,7 +94,11 @@
       id: "likelihoodAssessment",
       label: {
         text: "Your assessment"
-      }
+      },
+      value: data.likelihoodAssessment,
+      errorMessage: {
+        text: "Enter your assessment"
```
Comment (theDustRoom) [inline server/views/record-of-oral/risk-assessment.njk:100]: Changed.

**Review by sarahyjja — APPROVED** [review]
coolio

**Pull request merged**